lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -289,8 +291,10 @@ def error_code_from(succeeded, response)
         unless succeeded
           if errors = response["errors"]
             errors.first.try(:[], "code")
+          elsif status = response["statusOutput"]["statusCode"]
```

Comment:
The first `if` is already in that position. I'd be fine adding it across the board though; kind of wondering why I don't see it in more places.